dotnet/src/SemanticKernel.Abstractions/Data/RecordAttributes/VectorStoreRecordVectorAttribute.cs

```diff
@@ -34,7 +34,7 @@ public VectorStoreRecordVectorAttribute(int Dimensions)
     /// <param name="Dimensions">The number of dimensions that the vector has.</param>
     /// <param name="IndexKind">The kind of index to use.</param>
     /// <param name="DistanceFunction">The distance function to use when comparing vectors.</param>
-    public VectorStoreRecordVectorAttribute(int Dimensions, string IndexKind, string DistanceFunction)
+    public VectorStoreRecordVectorAttribute(int Dimensions, string? IndexKind, string? DistanceFunction)
```

Comment:
@westey-m changed to nullable because otherwise you can't cleanly set dimensions and distance, but skip index kind, when using attribute. (while it was perfectly fine to do using `VectorStoreRecordDefinition` properties).

Reply:
Sounds good.  It might make sense for me to swap these two params actually.  I imagine it would be more likely for someone to configure DistanceFunction than IndexKind.  I'll look into it after you have merged.